### ISSUE TITLE
Accessibility - add title to iframes

### ISSUE THREAD

**Issue by Ardenexal** [braintree/braintree-web#374]
### General information

* SDK version: 3.6.3
* Environment: Both
* Browser and OS : All

### Issue description
As per the Aria specification, all iframes should have a title to allow screen readers to describe what the content is.

https://dequeuniversity.com/rules/axe/3.0/frame-title

**Comment by crookedneighbor** [comment]
Thanks for the input, we'll look into this.

**Comment by crookedneighbor** [comment]
This is fixed in v3.35.0
